application/api/src/main/resources/application.yml

```
@@ -1,56 +1,29 @@
-spring.application.name: core-api
-spring.profiles.active: local
-
+# default
 spring:
-  config:
-    import:
-      - monitoring.yml
-      - logging.yml
-      - db-core.yml
+  profiles:
+    default: local
+  application:
+    name: api
   mvc.throw-exception-if-no-handler-found: true
   web.resources.add-mappings: false
-  security:
-    oauth2:
-      client:
-        provider:
-          naver:
-            authorization_uri: https://nid.naver.com/oauth2.0/authorize
-            token_uri: https://nid.naver.com/oauth2.0/token
-            user-info-uri: https://openapi.naver.com/v1/nid/me
-            user_name_attribute: response
-        registration:
-          naver:
-            client-id: f0Y2iXBYxDsBPH699BkC
-            client-secret: xxOOW3Nr6X
-            redirect-uri: http://localhost:8080/login/oauth2/code/naver
-            authorization-grant-type: authorization_code
-            scope:
-              - email
-              - name
-            client-name: naver
-          google:
-            client-id: ${GOOGLE-CLIENT-ID}
-            client-secret: ${GOOGLE-CLIENT-SECRET}
-            scope:
-              - email
-              - profile
-
----
-spring.config.activate.on-profile: local
-
-
----
-spring.config.activate.on-profile: local-dev
-
 
 ---
-spring.config.activate.on-profile: dev
-
+# local
+spring:
+  profiles:
+    group:
+      local: "security, db-core, logging, metrics"
```

```suggestion
      local: "security, db-core, logging, metrics"
      local: 
           - security
           - db-core
           - logging
           - metrics
```

별건 아니지만 이렇게도 관리가 가능할까요?
나중에 추가하고 변경할때 개행해서 사용하는게 더 좋을거 같아서요 ㅎㅎ

---

확인해보니 제안주신 방법으로도 관리가 가능해서 변경해두었습니다 ㅎㅎ
말씀주신 방법이 추가/삭제할 때 더 편리할 것 같네요! 👍